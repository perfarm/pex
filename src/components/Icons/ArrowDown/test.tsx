import React from 'react';

import { render } from '@testing-library/react';

import { ArrowDown } from '.';

describe('ArrowDown', () => {
  test('should match snapshot', () => {
    const { container } = render(<ArrowDown />);

    expect(container).toMatchSnapshot();
  });
});
