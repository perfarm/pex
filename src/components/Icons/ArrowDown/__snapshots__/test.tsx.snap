// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArrowDown should match snapshot 1`] = `
<div>
  <svg
    class="PJLV"
    fill="none"
    height="18"
    viewBox="0 0 16 18"
    width="18"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      class="PJLV"
      d="M8.19922 0.75V17.25M8.19922 17.25L1.44922 10.5M8.19922 17.25L14.9492 10.5"
      stroke="#898989"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="1.5"
    />
  </svg>
</div>
`;
